WcaOnRails/app/models/competition.rb

```diff
@@ -331,6 +331,20 @@ def warnings_for(user)
       unless has_any_round_per_event? && schedule_includes_rounds?
         warnings[:schedule] = I18n.t('competitions.messages.schedule_must_match_rounds')
       end
+
+      warnings = championship_warnings.merge(warnings)
```

Comment:
To be consistent with the handling of other warnings, shouldn't this go into an `if...end` block and be `warnings[:championships]` or something alike?

Reply:
There are two reasons that I broke with consistency from the above:

-I broke it into a separate function (`championship_warnings`) so that the WCAT email could only include these, and not other warnings.
-The key is the championship type rather than `:championships` because of the possibility of multiple (as we're discussing below)

I guarded it with an `if...end` for clarity.